kuka_kr3_support/urdf/kr3r540_macro.xacro

```diff
@@ -0,0 +1,171 @@
+        <geometry>
+          <mesh filename="package://kuka_kr3_support/meshes/kr3r540/visual/link_2.dae" />
+        </geometry>
+        <xacro:material_kuka_ral_pure_white />
+      </visual>
+      <collision>
+        <origin xyz="0 0 0" rpy="0 0 0" />
+        <geometry>
+          <mesh filename="package://kuka_kr3_support/meshes/kr3r540/collision/link_2.stl" />
+        </geometry>
+      </collision>
+    </link>
+    <link name="${prefix}link_3">
+      <visual>
+        <origin xyz="0 0 0" rpy="0 0 0" />
+        <geometry>
+          <mesh filename="package://kuka_kr3_support/meshes/kr3r540/visual/link_3.dae" />
+        </geometry>
+        <xacro:material_kuka_ral_pure_white />
+      </visual>
+      <collision>
+        <origin xyz="0 0 0" rpy="0 0 0" />
+        <geometry>
+          <mesh filename="package://kuka_kr3_support/meshes/kr3r540/collision/link_3.stl" />
+        </geometry>
+      </collision>
+    </link>
+    <link name="${prefix}link_4">
+      <visual>
+        <origin xyz="0 0 0" rpy="0 0 0" />
+        <geometry>
+          <mesh filename="package://kuka_kr3_support/meshes/kr3r540/visual/link_4.dae" />
+        </geometry>
+        <xacro:material_kuka_ral_pure_white />
+      </visual>
+      <collision>
+        <origin xyz="0 0 0" rpy="0 0 0" />
+        <geometry>
+          <mesh filename="package://kuka_kr3_support/meshes/kr3r540/collision/link_4.stl" />
+        </geometry>
+      </collision>
+    </link>
+    <link name="${prefix}link_5">
+      <visual>
+        <origin xyz="0 0 0" rpy="0 0 0" />
+        <geometry>
+          <mesh filename="package://kuka_kr3_support/meshes/kr3r540/visual/link_5.dae" />
+        </geometry>
+        <xacro:material_kuka_ral_pure_white />
+      </visual>
+      <collision>
+        <origin xyz="0 0 0" rpy="0 0 0" />
+        <geometry>
+          <mesh filename="package://kuka_kr3_support/meshes/kr3r540/collision/link_5.stl" />
+        </geometry>
+      </collision>
+    </link>
+    <link name="${prefix}link_6">
+      <visual>
+        <origin xyz="0 0 0" rpy="0 0 0" />
+        <geometry>
+          <mesh filename="package://kuka_kr3_support/meshes/kr3r540/visual/link_6.dae" />
+        </geometry>
+        <xacro:material_kuka_pedestal />
+      </visual>
+      <collision>
+        <origin xyz="0 0 0" rpy="0 0 0" />
+        <geometry>
+          <mesh filename="package://kuka_kr3_support/meshes/kr3r540/collision/link_6.stl" />
+        </geometry>
+      </collision>
+    </link>
+    <!-- This frame corresponds to the $FLANGE coordinate system in KUKA KRC controllers. -->
+    <link name="${prefix}tool0" />
+
+    <joint name="${prefix}joint_a1" type="revolute">
+      <origin xyz="0 0 0.345" rpy="0 0 0"/>
+      <parent link="${prefix}base_link"/>
+      <child link="${prefix}link_1"/>
+      <axis xyz="0 0 -1"/>
+      <limit lower="${-170 * DEG2RAD}" upper="${170 * DEG2RAD}" effort="0" velocity="${530 * DEG2RAD}"/>
```

Comment:
Now that Jade+ xacro is enabled for this macro (in 33ab5a7), we can use `${radians(..)}` expressions here. That would remove the need to use the custom `DEG2RAD` constant from `common_constants.xacro`.

For an example, see:

https://github.com/ros-industrial/kuka_experimental/blob/f84604221a8738f2e06d2268aed505a03d0c6537/kuka_kr150_support/urdf/kr150_2_macro.xacro#L123